ci/gpu/build.sh

```diff
@@ -51,7 +51,8 @@ conda install -c nvidia/label/cuda$CUDA_REL -c rapidsai/label/cuda$CUDA_REL -c r
       dask \
       distributed \
       dask-cudf=${MINOR_VERSION} \
-      dask-cuda=${MINOR_VERSION}
+      dask-cuda=${MINOR_VERSION} \
+      pyarrow=0.14.1.*
```

Comment:
I'm curious if you needed to install this to get cugraph to build/run tests, or is this just assumed to be needed?  If this is a build dependency, it'll also need to be in the `cpu/build.sh` script.

Reply:
Actually, I got this recommendation from @dillon-cullinan, he told me that this is necessary to pull out the most recent version of cuDF (before adding this, we were getting an older version of cuDF).

Reply:
@seunghwak this issue was fixed recently. pyarrow can now be removed from the install since we deleted the older packages from our conda channel that had outdated dependencies. I checked a fresh install without the pyarrow component and it grabbed the latest version from our channel.